src/SOS/extensions/hostcoreclr.cpp

```diff
@@ -68,6 +70,77 @@ bool g_hostingInitialized = false;
 static LPCSTR g_hostRuntimeDirectory = nullptr;
 static ExtensionsInitializeDelegate g_extensionsInitializeFunc = nullptr;
 
+struct RuntimeVersion
+{
+    uint32_t Major;
+    uint32_t Minor;
+};
+
+struct RuntimeHostingConstants
+{
+    constexpr static RuntimeVersion SupportedHostRuntimeVersions[] = {
+        {5, 0},
+        {3, 1},
+        {6, 0}
```

Comment:
What about 7.0? I think we will need to support 7.0 only installed.

Reply:
I didn't want to modify that in this PR this was the existing probing order, but it might not be a bad idea to do

6.0 -> 3.1 -> 5.0 -> 7.0

Reply:
Yes, I like that.